system/datatypes/peer_reviewed_article.php

```diff
@@ -1048,12 +1051,27 @@ private function getCommentFormHtml($type, DocPage $discussion = null) {
 		$comment = __('art-comment');
 		$new = $discussion === null;
 		$commentFieldName = self::FIELD_NAME_DISCUSSION_COMMENT . '/' . ($new ? 'new_' . $type : $discussion->getId());
-		$html = <<<EOF
+		if (strpos($commentFieldName, 'new_review') > 0) {
```

Comment:
Even though using strpos here works and is concise, it does not seem a good practice to me. I believe you want to say `if ($new && $type == 'review')` here, so probably best to just write that down. IMO that gives more readable code, and prevents problems later on when changing the field names without realizing this check must also be changed.

It is probably also *faster*, since strpos has to scan through the entire string until it finds the string you look for.